### ISSUE TITLE
[Feature request] Go to searched text in current file using in-file search

### ISSUE THREAD

**Issue by DrCaesarsPalaceMD** [scambier/obsidian-omnisearch#360]
When searching in-file, the popup window finds the results in the current file as intended, but selecting the search result with enter or a click does not scroll the cursor to the result part of the document. 

It would be great to have this feature added so the cursor jumps to the search result in the open file.

**Comment by scambier** [comment]
Is it a duplicate of #222?

**Comment by DrCaesarsPalaceMD** [comment]
> Is it a duplicate of #222?

Not quite, I read that issue as the search panel not opening at all. I am using live-preview mode, and the panel does open, it is possible to search. But the cursor doesn't scroll to the search results when selecting it.

**Comment by scambier** [comment]
I meant #173, but if it happens in Live Preview, it's a bug. Does it happen all the time, and what is your version of Omnisearch?

**Comment by DrCaesarsPalaceMD** [comment]
Ah, yeah it's not in reading mode, just live-preview. I've just tried and it's the same in source mode. I can replicate in about ten random notes in live-preview and source modes.

Omnisearch version 1.21.1
Obsidian v 1.5.11
MacOS
Other plugins:

 ---... see below.

Actually in just testing switching each plugin on and off, it's an issue with having the plugin Open In New Tab (v 1.0.9). With that plugin disabled it works exactly as intended, but with it enabled, cursor doesn't move when selecting the search result or cmd-enter.

So a quick fix is to disable Open In New Tab.
